app/views/static_pages/about.html.erb

```diff
@@ -1,10 +1,7 @@
-<% provide(:title, "About") %>
 <!DOCTYPE html>
 <html>
   <head>
-    <title>
-      <%= yield(:title) %> | Ruby on Rails Tutorial Sample App
-    </title>
+    <title><%= full_title(yield(:title)) %></title>
```

Comment:
I would think you can pass the title from these views up to the application layout without having to duplicate the `<head>` tag in each view. You should be able to just `provide(:title, 'About')` in this view. This file would only need the view's content:

```rb
<% provide(:title, "About") %> 
<h1>About</h1>
... the rest of the content ...
```

Reply:
Hey, I see what you are saying. The ```<body><%= yield %></body>``` in the application.html.erb's file is the placeholder for the HTML that gets injected into its position for whichever view is being accessed. Here the full_title method then interpolates the page title into the remaining base title, and none of the other HTML structures like head and body are needed because of where the yield is, if I understand correctly. It seems to be working beautifully after those changes. I will be creating a PR from a new branch called filling-in-the-layout shortly. @jkronz